### DISCUSSION TITLE
How to delete imported routing-files?

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#13590] [Q&A]
I just played a little bit about routing-profiles. But how can I delete all these imported profiles? I cant find anything. 

All XML-Files, from which  imported, I already deleted in folder "routing"

Useability is not so good sometimes in OSMAnd ;(
![Screenshot_20220120-143014](https://user-images.githubusercontent.com/62500258/150348626-4e53d344-4dd1-4e35-8e18-e3d64f258287.png)
(

**Comment by vshcherb** [comment]
If you have access to the files, they are stored in folder routing/.... You can migrate data to External Storage to get access.

**Comment by ThomasMeister123** [comment]
I have access. Because I put my own files in this folder. The question was, how can I delete the importet Profiles (WITHIN OSMAnd) [not the files!]

And: Wya OSMAnd make a copy of the file? When I put the file PROFILE.xml in the folder and import that in OSMAnd, the OSMAnd makes a copy (PROFILE 1.xml) and imported the copy. So, after that I have TWO files in the folder: PROFILE.xml and PROFILE 1.xml
